packages/components/src/snackbar/README.md

```diff
@@ -36,17 +36,54 @@ const MySnackbarNotice = () => (
 );
 ```
 
-#### Props
+### Props
 
 The following props are used to control the display of the component.
 
--   `children`: (string) The displayed message of a notice. Also used as the spoken message for assistive technology, unless `spokenMessage` is provided as an alternative message.
--   `spokenMessage`: (string) Used to provide a custom spoken message in place of the `children` default.
```

Comment:
I changed the type of `spokenMessage` from `string` to `ReactNode`.

The type `ReactNode` includes `string`.

And the fact that [children is the default of spokenMessage](https://github.com/kienstra/gutenberg/blob/62987d7e995a6d24f6b7d5a491c73aff5a1097c8/packages/components/src/snackbar/index.js#L46) was a problem.

```jsx
function Snackbar(
	{
		className,
		children,
		spokenMessage = children,
```

The type of `children`, `ReactNode`, isn't assignable to `string`:
<img width="861" alt="Screen Shot 2022-11-01 at 4 34 06 PM" src="https://user-images.githubusercontent.com/4063887/199355005-f435738f-70a3-46bf-8a37-559e83040dc8.png">

Reply:
Thanks, https://github.com/WordPress/gutenberg/pull/45472/commits/7b0c570fac9e8c67b41c3b32752e5f710d77188b changes both to `string`.